MyClient.ts

```diff
@@ -61,8 +32,8 @@ class MyClient {
 
   constructor(seed: string) {
     this.seed = seed;
-    this.collectionId = 735;
-    this.address = "5FNTBngp5E57ti1RYz7taHMChiqMvK2rQrSidH8nWwp1ALKW";
+    this.collectionId = -1;
+    this.address = "5FyM6ZTLNKQniseGKUwcqCL9hxoCGka62xV7ouTQ94afdqG5";
```

Comment:
we need to calculate it from seed, added to TODO

Reply:
upd: required changes, so - while u are here, fix it too pls

Reply:
done

Reply:
done